sphinx/ext/autosummary/generate.py

```diff
@@ -327,7 +327,7 @@ def get_modules(obj: Any) -> Tuple[List[str], List[str]]:
         ns['exceptions'], ns['all_exceptions'] = \
             get_members(obj, {'exception'}, imported=imported_members)
         ns['attributes'], ns['all_attributes'] = \
-            get_module_attrs(ns['members'])
+            get_module_attrs(dir(obj))
```

Comment:
This change puts the imported components to `all_attributes`.
In your example, `nx` will be added into `all_attributes`. I think it's not good behavior.

Reply:
@tk0miya  in the example, i want document `module attributes` show `a` and `G`,but now only can show Built-in Types(`a`) now. In `class attributes` can show `a` and `G` but `module attributes` can not show `a` and `G`.

## that can show `a` and `G` on `module attributes`
![image](https://user-images.githubusercontent.com/25873606/117537983-ad785880-b036-11eb-8338-3ab0ac9f6bf1.png)
## click the`G` i can see the detail
![image](https://user-images.githubusercontent.com/25873606/117538030-d1d43500-b036-11eb-8bb5-7d6d7818fef9.png)

Reply:
I said this implementation is not good because it causes regression error. So you need to change the way to realize your request.